test/e2e/pods.go

```diff
@@ -124,16 +124,14 @@ var _ = Describe("Pods", func() {
 		}
 
 		By("submitting the pod to kubernetes")
+		// We call defer here in case there is a problem with
```

Comment:
I'm not sure this comment is really necessary since the semantics of defer are well known to go developers.